fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java

```diff
@@ -680,6 +697,7 @@ public void write(DataOutput out) throws IOException {
         Text.writeString(out, exportPath);
         Text.writeString(out, columnSeparator);
         Text.writeString(out, lineDelimiter);
+        Text.writeString(out, columns);
```

Comment:
In fact, there is no need to modify the logic here. You only need to initialize the columns after reading the properties. 
Also, even if columns are to be persisted, they cannot actually be placed in this position.